$NetBSD: patch-panels_meson.build,v 1.1 2020/11/10 18:22:23 nia Exp $

user-accounts panel wants krb5.pc, this isn't provided by Heimdal.

Let's not break completely when Heimdal is the default krb5
implementation.

--- panels/meson.build.orig	2020-10-03 22:47:30.424089700 +0000
+++ panels/meson.build
@@ -14,19 +14,16 @@ panels = [
   'location',
   'lock',
   'microphone',
-  'mouse',
   'notifications',
   'online-accounts',
   'power',
   'printers',
-  'region',
   'removable-media',
   'search',
   'sharing',
   'sound',
   'universal-access',
-  'usage',
-  'user-accounts'
+  'usage'
 ]
 
 if host_is_linux
